test/fuzzer/pedro/aggregate_assertion_errors.test

```diff
@@ -31,4 +31,4 @@ SELECT count(c0 ORDER BY 0) FROM (SELECT 2 EXCEPT SELECT 2) c0;
 query I
 SELECT mode((c0, 0)) FROM (SELECT 1 c0), (SELECT 2);
 ----
-{'c0': 1, 'v2': 0}
+{'c0': 1, '': 0}
```

Comment:
I would say that `()` should always create an unnamed struct. That's the behavior in Postgres.

Perhaps we can also change the `ToString` of unnamed structs to leave out the name entirely, although that might be better done in a future PR.

Reply:
I also thought about the ToString modification, but that would break roundtripping of the type string, which is something to consider